capstone/src/main/webapp/dashboard-loader.js

```diff
@@ -0,0 +1,21 @@
+document.body.prepend(dynamicButton);
+
+function createClassroom() {
+    var auth2 = gapi.auth2.getAuthInstance();
+    var name = "";
+    var email = "";
+    if (auth2.isSignedIn.get()) {
+        name = auth2.currentUser.get().getBasicProfile().getName();
+        email = auth2.currentUser.get().getBasicProfile().getEmail();
+    }
+
+    var subject = document.querySelector('#subject').value;
```

Comment:
When you are running document.X, it is more clear to use the getElementById function than the general purpose querySelector (and more efficient since that has to run the CSS parser and may bring back potentially any number of nodes)

Reply:
Sounds good

Reply:
got it